src/components/ebay-radio/index.js

```diff
@@ -1,14 +1,17 @@
 
 module.exports = require('marko-widgets').defineComponent({
     template: require('./template.marko'),
-    handleClick(originalEvent) {
-        const target = originalEvent.target;
+    handleClick: forwardEvent('change'),
+    handleFocus: forwardEvent('focus')
+});
 
-        if (!target.disabled) {
-            this.emit('radio-change', {
+function forwardEvent(eventName) {
+    return function(originalEvent, el) {
+        if (eventName !== 'change' || (eventName === 'change' && !el.disabled)) {
```

Comment:
Why not do `if (eventName !== 'change || !el.disabled)`.
That way if eventName is change and it is disabled then it won't pass. Otherwise if event is not change it'll pass or if the el is not disabled. 

Also even if they can't focus, this seems like extra unnecessary logic, I would say if we're going with this route, then a simple `if (!el.disabled)` should be fine. It's not like they're going to trigger this event on a disabled radio anyways

Reply:
Agree. Let's keep it consistent with checkbox, I will remove the disabled check.

Reply:
Don't remove the disabled check, it is needed. 
Add it for both cases imo